pingviewer.pro

```diff
@@ -24,11 +24,12 @@ RESOURCES += \
 
 # Warning as error
 *-g++ | *-clang {
-    QMAKE_CXXFLAGS += -Werror
+    QMAKE_CXXFLAGS += -Werror -Os
```

Comment:
this is optimization flag for code size

> It also enables -finline-functions, causes the compiler to tune for code size rather than execution speed

https://gcc.gnu.org/onlinedocs/gcc-9.2.0/gcc/Optimize-Options.html#Optimize-Options

Reply:
It appears that Ofast was lost somewhere in the rebase, fixed.